pom.xml

```diff
@@ -5,11 +5,11 @@
         <!-- https://mvnrepository.com/artifact/org.jenkins-ci.plugins/plugin -->
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>4.10</version>
+        <version>4.55</version>
```

Comment:
Please change the version here to 4.54 or 4.56.
I remember one version was buggy.